package.json

```diff
@@ -34,6 +34,7 @@
 		"dotenv-cli": "^7.4.2",
 		"mysql2": "^3.10.1",
 		"react-error-boundary": "^4.0.13",
+		"recharts": "^2.12.7",
```

Comment:
Das sollte meiner Meinung nicht auf der Hauptebene passieren. Entweder in apps/web installieren oder das Chart selbst verschieben (s. https://github.com/pgenergy/energyleaf/pull/140/files#r1685328263)